liferay-content-management/liferay-portal
# LPD-32874 Add resource command for autosave

LPD-32874 Add resource command for autosave

# What is this trying to solve?

- Add resource command for autosave feature to better performance and manage errors.
- It solves a bug: Before this fix, we coulnd't publish a Web Content when editing it as autosave does not update the version of the article.

[Link to ticket](https://liferay.atlassian.net/browse/LPD-32874)

# How am I fixing it?

- @Mikellorza Added a Resource Command to retrieve the data we need to update for the article after teh autosave.
- Then, I added some steps to update teh article version, status and ID after autosaving.

# How can I test it?

- Enable FF 'LPD-11228': true,
		'LPD-15596': true,
- Create a WC
- Fill the title 
- Wait for the Autosave indicator
- Assert version, draft and article ID appears
- Publish WC
- Edit the WC
- Make some chnage in the title
- Assert version changes and draft status label appears

https://github.com/user-attachments/assets/bc0dc524-378a-4daa-a14a-705d77af67c8
